cabal.project

```diff
@@ -9,6 +9,8 @@ packages: .
 
 jobs: $ncpus
 
+allow-newer: all:template-haskell
```

Comment:
This will go away when we bump `tree-sitter-php` (and any other packages that have a too-restrictive TH upper bound).